src/components/DotaMap/index.jsx

```diff
@@ -9,15 +10,26 @@ const setMapSizeStyle = (width, maxWidth) => ({
   maxHeight: maxWidth,
 });
 
-// 12/13/2016 @ 5:00pm (UTC)
-const isPost700 = unixDate => unixDate > 1481648400;
+const dotaMaps = [
+  { patchName: '7.07', mapImage: '/assets/images/dota2/map/detailed_707.png' },
+  { patchName: '7.00', mapImage: '/assets/images/dota2/map/detailed_700.png' },
+  { patchName: '6.86', mapImage: '/assets/images/dota2/map/detailed_686.png' },
+  { patchName: '6.82', mapImage: '/assets/images/dota2/map/detailed_682.png' },
+  { patchName: '6.70', mapImage: '/assets/images/dota2/map/detailed_pre682.png' },
+];
+
+const defaultMap = '/assets/images/dota2/map/detailed.png';
```

Comment:
I think this is the pre_700 map from before.  Now we have more accurate maps.

I think the default should be the latest map `dotaMaps[0].mapImage`, and we should delete detailed.png

Reply:
Okay, sounds good

Reply:
Done